canonical/lxd-ui
# Update images

## Done

- changed image order to be ubuntu images from newest to oldes with lts on top
- added image update script
- updated images (in draft, because of the pending release of lunar lobster)

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - check image list for creating instances
